docs/build/manifest/cosmos.md

```diff
@@ -165,12 +165,12 @@ The following table explains filters supported by different handlers.
 
 **Your SubQuery project will be much more efficient when you only use `TransactionHandler`, `MessageHandler`, or `EventHandler` handlers with appropriate mapping filters (e.g. NOT a `BlockHandler`).**
 
-| Handler                                                       | Supported filter          |
-| ------------------------------------------------------------- | ------------------------- |
-| [cosmos/BlockHandler](./mapping/polkadot.md#block-handler)             | `modulo`                  |
-| [cosmos/TransactionHandler](./mapping/polkadot.md#transaction-handler) | No filters                |
-| [cosmos/MessageHandler](./mapping/polkadot.md#message-handler)         | `type`, `values`\*        |
-| [cosmos/EventHandler](./mapping/polkadot.md#event-handler)             | `type`, `messageFilter`\* |
+| Handler                                                       | Supported filter                               |
+| ------------------------------------------------------------- | ---------------------------------------------- |
+| [cosmos/BlockHandler](./mapping/polkadot.md#block-handler)             | `modulo`                              |
+| [cosmos/TransactionHandler](./mapping/polkadot.md#transaction-handler) | `includeFailedTx`                     |
+| [cosmos/MessageHandler](./mapping/polkadot.md#message-handler)         | `includeFailedTx`, `type`, `values`\* |
```

Comment:
Im not sure what the `\*` on these 2 lines. They don't seem to refer to anywhere

Reply:
Looks outdated